suite/tests/CMakeLists.txt

```diff
@@ -6579,7 +6579,7 @@ IF (NOT ANDROID AND NOT APPLE AND NOT MUSL AND NOT RISCV64 AND NOT WINDOWS)
       "${PROJECT_BINARY_DIR}/${INSTALL_BIN}/drsyscall_record_trimmer@"
       "-input_file@${syscall_record_test_file}@"
       "-output_file@${syscall_record_test_file}.trimmed@"
-      "-trim_before_timestamp@10013@-trim_after_timestamp@10014")
+      "-trim_before_timestamp@10013@-trim_after_timestamp@10015")
```

Comment:
Setting -trim_before_timestamp to 10012 should keep the same set of records in the output? And it would also test for the partial-record-at-beginning  case?

Reply:
Do you mean it won't trim records at the beginning? But the records after timestamp 10011 till the next start timestamp 10013 would be trimmed right?